src/common/components/ListAllContacts.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import {
+  Image,
+  StyleSheet,
+  Text,
+  View,
+  FlatList,
+  TouchableOpacity,
+} from 'react-native';
+
+import {useNavigation} from '@react-navigation/native';
+
+import {allContacts} from '../../api/fakeApiUser';
+import routeConstants from '../../utils/constants/routeConstants';
+import typography from '../../themes/typpography';
+import {colors} from '../../themes/theme';
+import {scale} from 'react-native-size-matters';
+
+const ListAllContacts = () => {
+  const navigation = useNavigation();
+
+  const handlePress = () => {
+    navigation.navigate(routeConstants.DETAILS);
+  };
```

Comment:
we can turn this into a one liner 
`const handlePress = () => navigation.navigate(routeConstants.DETAILS)`